gradle.properties

```diff
@@ -1,4 +1,4 @@
-PROJECT_VERSION=1.1.0
+PROJECT_VERSION=1.2.0-SNAPSHOT
```

Comment:
yes, if they want to get the pre-release version. latest release is 1.1.0 but current master is 1.2.0-SNAPSHOT (pre-release) and next release will be 1.2.0. Once we release 1.2.0, this will be 1.3.0-SNAPSHOT (or whatever the next version is gonna be), and so on